### ISSUE TITLE
Current issues/improvements being handled

### ISSUE THREAD

**Issue by lokinmodar** [lokinmodar/Echoglossian#67]
- Cutscenes overlays (TalkSubtitle) and DB saving
- Translate Chat Bubbles (Minitalk on NPCs)
- Fix duplications in DB saving
- Fix BattleTalk
- Async handling of UI elements

**Comment by ghost** [comment]
does adjusting the async handling from UI solves the stuttering problem when translating?

**Comment by lokinmodar** [comment]
> does adjusting the async handling from UI solves the stuttering problem when translating?

Yes! This is my current work in progress. Not finished yet but almost there https://github.com/lokinmodar/Echoglossian/issues/72

**Comment by lokinmodar** [comment]
Closing this old meta-tracker.

This kind of broad work-in-progress tracking is now better handled by the changelog, current docs, and focused issues for the remaining gaps instead of keeping a catch-all issue open.
